src/containerapp/azext_containerapp/commands.py

```diff
@@ -201,3 +201,8 @@ def load_command_table(self, _):
         g.custom_command('add', 'add_workload_profile')
         g.custom_command('update', 'update_workload_profile')
         g.custom_command('delete', 'delete_workload_profile')
+
+    with self.command_group('containerapp patch', is_preview=True) as g:
+        g.custom_command('list', 'patch_list', is_preview=True)
```

Comment:
remove is_preview=True in command when there is one in command group

Reply:
Thanks @Juliehzl, the patch command has list, apply and interactive. Could you please clarify if `remove is_preview=True` needs to be removed from the group or each command?

Reply:
Updated.